numpy/linalg/linalg.py

```diff
@@ -33,6 +34,28 @@
 from numpy.lib.twodim_base import triu, eye
 from numpy.linalg import _umath_linalg
 
+from numpy._typing import NDArray
+
+class EigResult(NamedTuple):
+    eigenvalues: NDArray
```

Comment:
```suggestion
    eigenvalues: NDArray[Any]
```
It's considered best practice to default to `Any` rather than leaving a generic parameter empty.